pkg/server/routes.go

```diff
@@ -44,23 +48,42 @@ func (h StaticHandler) ServeHTTP(ctx *gin.Context) {
 	http.FileServer(h.Assets).ServeHTTP(ctx.Writer, ctx.Request)
 }
 
-// SetupRoutes registers handlers for various URL paths. You can call this
-// function more than once but only the first call will have an effect.
+// SetupRoutes registers handlers for various URL paths.
 func (srv *RegistrationServer) SetupRoutes() error {
 	var err error
 	srv.routesSetup.Do(func() {
+		// initialize default managers
+		keyManager, err := auth.InitializeDefaultKeyManager(srv.Logger(), srv.Config())
+		if err != nil {
+			err = errs.Wrapf(err, "failed to init default key manager: %s", err.Error())
```

Comment:
Every error here is fatal. We should immediately abort and fail/exit. So, if something is wrong during setting up the routs the service should fail to initialize and the k8s deployment will fail too.
Just adding `return` here should be enough I think. Otherwise `err` can be overridden later (in the line 60 for example) and become `nil` again.

Reply:
Fixed d4c84a5.